Document the workaround for docker for mac and Infinispan (#158)
quarkusio/quarkus-quickstarts: infinispan-client/README.md

# Start the Infinispan server

You have two options:

Option 1: Running with Docker `docker run -it -p 11222:11222 jboss/infinispan-server:latest`

> If you are using Docker For Mac, there is an known issue. Blog post [here](https://blog.infinispan.org/2018/03/accessing-infinispan-inside-docker-for.html)

Reviewer: ```suggestion
If you are using Docker For Mac, there is a known issue (see [this blog post](https://blog.infinispan.org/2018/03/accessing-infinispan-inside-docker-for.html for more details).
```

You probably will drop it if you include the `hotrod-client.properties` but who knows :).

Reviewer: Ah so I see you included the file by default. Maybe we could drop the instructions then?

Reviewer 2: +1 to drop the instructions here

Author: Maybe just reword? To explain that it is not mandatory if you are not using docker 4 mac

Author: I proposed a rewording. I prefer to explain why this file is there to avoid confusion around it. WDYT?
